Mage/src/main/java/mage/cards/Card.java

```diff
@@ -133,13 +134,13 @@
 
     void addAbility(Ability ability);
 
-    boolean addCounters(Counter counter, Ability source, Game game);
+    boolean addCounters(Counter counter, Controllable source, Game game);
```

Comment:
Why you changed param to Controllable interface and do not use that? Cards are public and changes in interface will broke the compatibility with old clients (@LevelX2 must be change base version on next release, see #4459).

Reply:
Alright, I'll see if I can avoid that issue.